test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -238,6 +239,15 @@ protected void Log(string fmt, params object[] args)
             Output.WriteLine(fmt, args);
         }
 
+        protected void LogConfigurationSpecificBaseline(string baselineConfigDir)
```

Comment:
So this is going to log for every file comparison? Can this be done only once per test?